specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -1291,6 +1443,22 @@
       "in": "path",
       "x-ms-skip-url-encoding": true
     },
+    "OcrLanguage": {
+      "name": "language",
+      "in": "query",
+      "description": "The BCP-47 language code of the text to be detected in the image.",
+      "required": true,
+      "x-ms-parameter-location": "method",
+      "type": "string",
+      "x-ms-enum": {
+        "name": "OcrDetectionLanguage",
```

Comment:
Reuse Language for the x-ms-enum name so we don't generate a new enum type in the SDK.  I have verified this in SDK generation.